aptos-move/framework/aptos-framework/sources/genesis.spec.move

```diff
@@ -9,7 +9,8 @@ spec aptos_framework::genesis {
     }
 
     spec initialize_for_verification {
-        pragma verify = true;
+        // TODO: disabled due to the issue of Table.
```

Comment:
Is there a github issue tracking this table issue we can link here?

Reply:
Yes, here is the github issue (https://github.com/move-language/move/issues/680) which @meng-xu-cs is looking at.